packages/swingset-xsnap-supervisor/README.md

```diff
@@ -36,12 +36,12 @@ assert.equal(bundle.moduleFormat, 'nestedEvaluate');
 await worker.evaluate(`(${bundle.source}\n)()`.trim());
 ```
 
-To help detect version drift or build/import problems, the package also exports the hex-encoded SHA256 hash of the JSON-stringified bundle. This should be identical to running `/usr/bin/shasum -a 256` on the pathname recorded in the internal `bundlePaths.supervisor` (typically `swingset-xs-supervisor/bundles/supervisor.bundle`). Clients which have selected a particular version of `@agoric/swingset-xs-supervisor` in their `package.json` can retrieve this hash at import time and compare it against a hard-coded copy, to assert that their preferred version is actually in use. Such clients would need to update their copy of the hash each time they deliberately switch to a different version.
+To help detect version drift or build/import problems, the package also exports a function to get the hex-encoded SHA256 hash of the JSON-stringified bundle. This should be identical to running `/usr/bin/shasum -a 256` on the pathname recorded in the internal `bundlePaths.supervisor` (typically `swingset-xs-supervisor/bundles/supervisor.bundle`). Clients which have selected a particular version of `@agoric/swingset-xs-supervisor` in their `package.json` can retrieve this hash at import time and compare it against a hard-coded copy, to assert that their preferred version is actually in use. Such clients would need to update their copy of the hash each time they deliberately switch to a different version.
 
 ```js
-import { supervisorBundleSHA256 } from '@agoric/swingset-xs-supervisor';
+import { getSupervisorBundleSHA256 } from '@agoric/swingset-xs-supervisor';
 const expected = '54434e4a0eb0c2883e30cc43a30ac66bb792bec3b4975bb147cb8f25c2c6365a';
-assert.equal(supervisorBundleSHA256, expected, 'somehow got wrong version');
+assert.equal(await getSupervisorBundleSHA256(), expected, 'somehow got wrong version');
```

Comment:
Ha I had suggested `readFileSync` to avoid forcing consumers to do this kind of top-level-await.

Reply:
So I used an async function for getting the bundle itself, to support a version of this package that builds a bundle on-demand (for the #7056 flavor). And then I realized that our get-the-hash function can't be any *more* synchronous than the get-the-bundle function, so I made it be async too.

OTOH, any client code that's using the -dev flavor is not going to be fetching the hash anyways. Probably.

Anyways, I couldn't think of a case where the caller would be unduly inconvenienced by needing to `await` the hash. We can always change it later if that turns out to be awkward.. I'll find out more as I make more progress on #6596.